### PR TITLE
Bxmsdoc 6227 master

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

**Review by spolti — APPROVED** [review]

File: assemblies/assembly-openshift-authoring-managed.adoc [hunk 1]
```diff
@@ -17,12 +17,10 @@ and other business assets.
 .Prerequisites
 
 * {OPENSHIFT} version 3.11 is deployed.
-* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
-* For a high-availability deployment, the following resources are available on the OpenShift cluster:
-** For the {CENTRAL} replicated pod, 8 gigabytes of memory and 2 CPU cores are required for each replica. Two replicas are created by default.
-** For the {KIE_SERVER} replicated pod, 1 gigabyte of memory and 1 CPU core are required for each replica. Two replicas are created by default.
-** The Red Hat AMQ replicated pod uses the default resource limits configured on your cluster.
-** For the Red Hat Data Grid replicated pod, 2 gigabytes of memory and 1 CPU core are required for each replica. Two replicas are created by default.
+* The following resources are available on the OpenShift cluster. Depending on the load, higher resource allocation might be necessary for acceptable performance.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-authoring-managed.adoc:20]: do you mean memory load?

File: assemblies/assembly-openshift-authoring-managed.adoc [hunk 2]
```diff
@@ -17,12 +17,10 @@ and other business assets.
...
+** For the {CENTRAL} pod, 4 gigabytes of memory and 2 virtual CPU cores. In a high-availability deployment, these resources are required for each replica and two replicas are created by default.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-authoring-managed.adoc:21]: Minor rewrite suggestion for both bullets:

, these resources are required for each replica. Two replicas are created by default.

File: assemblies/assembly-openshift-authoring.adoc [hunk 3]
```diff
@@ -17,13 +17,10 @@ and other business assets.
...
 
 * {OPENSHIFT} version 3.11 is deployed.
-* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
-* For a high-availability deployment, the following resources are available on the OpenShift cluster:
-** For the {CENTRAL} replicated pod, 8 gigabytes of memory and 2 CPU cores are required for each replica. Two replicas are created by default.
-** For the {KIE_SERVER} replicated pod, 1 gigabyte of memory and 1 CPU core are required for each replica. Two replicas are created by default.
-** For the Red Hat Data Grid replicated pod, 2 gigabytes of memory and 1 CPU core are required for each replica. Two replicas are created by default.
-** The Red Hat AMQ replicated pod uses the default resource limits configured on your cluster.
-** The MySQL replicated pod uses the default resource limits configured on your cluster.
+* The following resources are available on the OpenShift cluster. Depending on the load, higher resource allocation might be necessary for acceptable performance.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-authoring.adoc:20]: Same comments as above

File: assemblies/assembly-openshift-freeform.adoc [hunk 4]
```diff
@@ -13,7 +13,9 @@ and other business assets. You can deploy any number of managed {KIE_SERVERS} an
 .Prerequisites
 
 * {OPENSHIFT} version 3.11 is deployed.
-* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
+* The following resources are available on the OpenShift cluster. Depending on the load, higher resource allocation might be necessary for acceptable performance.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-freeform.adoc:16]: memory load?

File: assemblies/assembly-openshift-freeform.adoc [hunk 5]
```diff
@@ -13,7 +13,9 @@ and other business assets. You can deploy any number of managed {KIE_SERVERS} an
...
+** 2 gigabytes of memory and 1 virtual CPU cores for each replica of the {CENTRAL} Monitoring pod.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-freeform.adoc:17]: core - singular

File: assemblies/assembly-openshift-immutable.adoc [hunk 6]
```diff
@@ -20,13 +20,11 @@ to add and update the business assets.
 .Prerequisites
 
 * {OPENSHIFT} version 3.11 is deployed.
+* The following resources are available on the OpenShift cluster. Depending on the load, higher resource allocation might be necessary for acceptable performance.
 ifdef::PAM[]
-* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
-** If you do not deploy monitoring infrastructure but only deploy an immutable {KIE_SERVER}, three gigabytes can be sufficient.
+** 2 gigabytes of memory and 1 virtual CPU cores for each replica of the {CENTRAL} Monitoring pod.
```
Comment (michelehaglund) [inline assemblies/assembly-openshift-immutable.adoc:25]: core - singular

**Review by michelehaglund — APPROVED** [review]
Looks good @mramendi Minor inline comments.

**Pull request merged**